tensorflow/python/kernel_tests/cast_op_test.py

```diff
@@ -146,9 +147,16 @@ def testInfNan(self):
     if sys.byteorder == "big":  
       self._compare(np.inf, np.int32, i4.max, False)  
       self._compare(np.inf, np.int64, i8.max, False)  
-    else:  
-      self._compare(np.inf, np.int32, i4.min, False)  
-      self._compare(np.inf, np.int64, i8.min, False)  
+    else:
+      #np.float64("np.inf").astype(np.int32) is negative on x86 but positive on ppc64le
+      #Numpy link to relevant discussion - https://github.com/numpy/numpy/issues/9040
+      #Tensorflow link to relevant discussion - https://github.com/tensorflow/tensorflow/issues/9360
+      if platform.machine() == "ppc64le":
+        self._compare(-np.inf, np.int32, i4.min, False)
```

Comment:
As I read the posts linked to above, -np.inf is technically undefined?  
I think this is a little ugly, but fine as a temporary fix, and it doesn't affect other architectures.